City-of-Pori/pori-kada
# Pori x fontawesome icon fix

To test: 
- Checkout this branch and clear the caches
- Go to https://local.pori.fi and see if the icons have returned
